fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,15 +457,82 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName"
+
+# --- Do slice time correction if indicated
```

Comment:
Can we add to this comment the following:
```
# Note that in the case of STC, $fMRIFolder/$OrigTCSName will NOT be the "original" time-series
# but rather the slice-time corrected version thereof.
```

Reply:
Added.